Sources/Orbit/Components/InputField.swift

```diff
@@ -37,25 +39,58 @@ public struct InputField: View {
             
             InputContent(
                 prefix: prefix,
-                suffix: suffix,
+                suffix: isSecure ? securedSuffix : suffix,
                 state: state,
                 message: message,
                 isEditing: isEditing,
-                suffixAction: suffixAction
-            ) {
-                HStack(spacing: 0) {
-                    textField
-                    Spacer(minLength: 0)
-                    clearButton
+                suffixAction: {
+                    if isSecure {
```

Comment:
The "eye" icon suffix will be specific to the InputField and should not clash with any existing suffix. It needs its own icon and action.

The full possible order of suffixes will be:
"X(clear value) - Eye(toggle password) - Suffix(any caller suffix)"

We may or may not want to show X(clear) button on secure fields - normally we do, so we do not have to change it for now.

Reply:
So that means I have to split this logic: eye is a standalone element, right?

Reply:
Yep, the eye icon would be an additional view that will be added (to the HStack of existing suffixes) only when `isSecture = true`.